guides/creating-your-first-micronaut-app/creating-your-first-micronaut-app.adoc

```diff
@@ -1,12 +1,4 @@
-= @guideTitle@
-
-@guideIntro@
-
-Authors: @authors@
-
-Micronaut Version: @micronaut@
-
-include::{commondir}/common-gettingStarted.adoc[]
+include::{commondir}/common-header.adoc[]
```

Comment:
```suggestion
include::{commondir}/common-header.adoc[]

include::{commondir}/common-gettingStarted.adoc[]
```

Reply:
Not sure I follow `common-header` already contains `common-gettingStarted`